code/mathematical-algorithms/exponentiation_power/exponents.py

```diff
@@ -0,0 +1,4 @@
+base = input("Give me the base")
```

Comment:
Since you are using python 2 for this code, use `raw_input` and cast it to an int, for catching all errors.